section-05-production-model-package/README.md

```diff
@@ -0,0 +1,4 @@
+# Deployment of Machine Learning Models
```

Comment:
I would leave the readme for later, I was thinkning of making a nicer one with logos

Reply:
I was going to use it to pull in a description in the setup.py, but yeah think it's not really adding much, removing